docker_signal.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
Seems that it should shut it down cleanly from what I'm reading. Just curious if you think otherwise?